flow/workflows/setup_flow.go

```diff
@@ -207,7 +207,7 @@ func (s *SetupFlowExecution) fetchTableSchemaAndSetupNormalizedTables(
 
 		tableNameSchemaMapping[flowConnectionConfigs.TableNameMapping[srcTableName]] = srcTableSchema
 
-		s.logger.Info("setting up normalized table for table %s for peer flow - ", srcTableSchema, s.PeerFlowName)
+		s.logger.Info("setting up normalized table for table %s for peer flow - %s", srcTableName, s.PeerFlowName)
```

Comment:
this has to be `Infof`

Reply:
doesn't appear to be present in the Temporal `log.Logger` type

Reply:
If it's info does it support `%s` ?

Reply:
doesn't support, changed to use fmt.Sprintf